lib/ClimaLandSimulations/src/Fluxnet/Fluxnet.jl

```diff
@@ -0,0 +1,64 @@
+module Fluxnet
+
+using ClimaLandSimulations
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using ClimaCore
+import CLIMAParameters as CP
+using Statistics
+using Dates
+using Insolation
+using StatsBase
+using Interpolations
+using ClimaLand
+using ClimaLand.Domains: Column
+using ClimaLand.Soil
+using ClimaLand.Soil.Biogeochemistry
+using ClimaLand.Canopy
+using ClimaLand.Canopy.PlantHydraulics
+import ClimaLand.Parameters as LP
+import ClimaComms
+context = ClimaComms.context()
+using Unitful: R, L, mol, K, kJ, °C, m, g, cm, hr, mg, s, μmol, Pa, W, mm, kPa
+using UnitfulMoles: molC
+using Unitful, UnitfulMoles
+using ArtifactWrappers
+using DelimitedFiles
+using Thermodynamics
+using Formatting
+using DataFrames
+
+# UnitfulMoles compound
+@compound CO₂
+
+# Directory paths
+climalandsimulations_dir = pkgdir(ClimaLandSimulations)
+climaland_dir = pkgdir(ClimaLand)
+
+# ClimaParameters
+include(joinpath(climaland_dir, "parameters", "create_parameters.jl"))
+const FT = Float64
+earth_param_set = create_lsm_parameters(FT)

```

Comment:
FT should be defined in the driver, not in src, so that we can run simulations with either Float32 or Float64. Could this setup be moved to fluxnet_simulation.jl?

Reply:
done!